boxsdk/object/task.py

```diff
@@ -0,0 +1,58 @@
+# coding: utf-8
+
+from __future__ import unicode_literals
+
+import json
+
+from .base_object import BaseObject
+from boxsdk.config import API
+from boxsdk.util.translator import Translator
+from ..pagination.marker_based_object_collection import MarkerBasedObjectCollection
+
+
+class Task(BaseObject):
+    """Represents a Box task."""
+
+    _item_type = 'task'
+
+    def assign(self, assign_to_id=None, assign_to_login=None):
+        url = self._session.get_url('task_assignments')
+        body = {
+            'task': {
+                'type': 'task',
+                'id': self.object_id
+            },
+            'assign_to': {
+                'id': assign_to_id,
```

Comment:
I assume that these are mutually exclusive, and should not both be specified at once.

I would suggest doing this:

Have just an `assign_to` parameter. It can be a `BaseObject` or a `unicode` string. If it is a `unicode` string, then set `assign_to.login`. Otherwise, set `assign_to.id` to `assign_to.object_id`.

(If the API allows you to pass `assign_to.type`, then for good measure, you should pass `assign_to.object_type` as `assign_to.type`, or else the `"user"` constant if you're passing a login.)